src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -165,17 +165,29 @@ private void LogMetric(IEnumerable<KeyValuePair<string, object>> values)
         }
 
         // Applies scope properties; filters most system properties, which are used internally
-        private static void ApplyScopeProperties(IDictionary<string, string> properties)
+        private static void ApplyScopeProperties(ITelemetry telemetry)
         {
-            var scopeProperties = DictionaryLoggerScope.GetMergedStateDictionaryOrNull();
+            var scopeProperties = DictionaryLoggerScope.GetMergedStateDictionaryOrNull();            
```

Comment:
nit: remove extra whitespace
```suggestion
            var scopeProperties = DictionaryLoggerScope.GetMergedStateDictionaryOrNull();
```